docs/data-sources/builtin-toolsets/github-mcp.md

```
@@ -7,7 +7,7 @@ The GitHub MCP server provides access to GitHub repositories, pull requests, iss
 Holmes supports two authentication methods for GitHub. Both deploy a self-hosted MCP server pod in your cluster that wraps the [official GitHub MCP server](https://github.com/github/github-mcp-server):
 
 - **Personal Access Token (PAT)**: Uses the standard `github-mcp` image. The PAT is passed directly to the MCP server.
-- **GitHub App**: Uses the `github-app-mcp` image which automatically generates and refreshes short-lived installation tokens from GitHub App credentials.
+- **GitHub App**: Uses the `github-app-mcp` image which automatically generates and caches short-lived installation tokens from GitHub App credentials. One App installed on several organizations can be served from a single deployment via [multi-organization support (alpha)](#multi-organization-support-alpha).
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)